tests/test_host_check.py

```diff
@@ -1327,7 +1327,7 @@ class TestLSMs(_CheckTestBase):
 
     check_group = "base"
```

Comment:
Have you collected code coverage? Can do so by:
- setting up a venv: `python3 -m venv venv` then `source venv/bin/activate`
- installing the coverage plugin: `pip install pytest-cov`
- run pytest, telling it to collect html code coverage: `pytest tests/ --cov=scripts/ --cov-report=html`
- view the coverage, in the htmlcov directory in vscode
Make sure all the lines of code you've added/updated are covered by the tests, and copy the htmlcov directory to your /ws/ a put a pointer to it here